lib/customer.rb

```diff
@@ -0,0 +1,40 @@
+require 'csv'
+require 'pry'
+
+# Wave 1 customer creations
+
+class Customer
+  attr_reader :id
+  attr_accessor :email, :address
+
+  @@customers = []
+
+  def initialize(id, email, address)
+    @id = id
+    @email = email
+    @address = address
+  end
+
+  #  Wave 2
+  # Load data from CSV
+  def self.all
+    @@customers = CSV.read("data/customers.csv").map do |line|
+      id = line[0].to_i
+      email = line[1]
```

Comment:
You could use a local variable here instead of a class variable, since you don't ever reuse this data outside of this method.